src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+/**
+ * Encapsulates the details of a Todo which is a subtype of Task.
+ */
+public class Todo extends Task {
```

Comment:
This is an add-on from my comment on how commands are implemented as enum. If commands are inherited from a `Command` interface, say `ToDoCommand`, then it would make more sense for this to be name as `ToDoTask`. What do you think?